build_tools/benchmarks/benchmark_helper.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
```

Comment:
[Here for threading]

I don't really follow what this is for exactly. What am I doing with this json that is being output? How is it different from the json in the exported benchmark configurations? Maybe it would help to share the documentation you wrote for how to locally run the benchmarks and then this would make more sense?

Reply:
So here are the examples I want to put in the doc [link](https://github.com/openxla/iree/blob/e4f41ff70e29931f09f0c88917a85cd1164212c6/build_tools/benchmarks/README.md#just-to-get-the-compile-and-run-flags). If we don't provide a tool, then users will have to walk through the serialized objects and their ids to get these flags ([examples](https://github.com/openxla/iree/blob/e4f41ff70e29931f09f0c88917a85cd1164212c6/build_tools/benchmarks/README.md#get-information-of-an-execution-benchmark)). It's kinda back to the problem of unfriendly serialization format.

How about making the flags outputted by this tool copy-pastable? So the usage of this tool is more straightforward

Reply:
But the copy-pastable flag will likely be a long string with whitespaces separating each flags. JSON doesn't support multline strings (and I really hope the output format is still JSON, so can work with jq). Windows and Linux shell also have different formats for multiline commands.

Reply:
I would recommend that we have output modes for just a full command line without json. No attempt to line wrap (terminals should do that for us). If people want to see a specific configuration, then something like this may be useful. Basically benchmark search functionality. So I could say "hey I want to see the mobile bert benchmark" and search for that, but then realize there are a bunch of those and narrow their search, etc. I don't really see the use case for wanting to dump everything just in a slightly rearranged format.

Reply:
SG. I changed it to just output command lines without JSON now. It's now also based on a small change #12565, which needs to be reviewed first.